index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ramiro Aquistapace</title>
+    <link rel="stylesheet" href="style.css" />
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <div class="wrapper">
+      <header class="header">
+        <a class="mylogo" href="#">My Logo</a>
+        <img alt="" src="Union.png" class="icon" />
```

Comment:
### OPTIONAL
- [ ] You can wrap this icon by `nav` tag.
- [ ]  Instead of using pictures for icons you can use [font-awesome-icons](https://fontawesome.com/start).